community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexAccessor.java

```diff
@@ -103,7 +103,7 @@ public IndexReader newReader()
     @Override
     public BoundedIterable<Long> newAllEntriesReader()
     {
-        BoundedIterable<Long>[] entries = instanceSelector.instancesAs( new BoundedIterable[INSTANCE_COUNT], IndexAccessor::newAllEntriesReader );
+        BoundedIterable[] entries = instanceSelector.instancesAs( new BoundedIterable[INSTANCE_COUNT], IndexAccessor::newAllEntriesReader );
```

Comment:
intentional change?

Reply:
Yes. It was messing with my IntelliJ for some reason claiming compilation issues, even though it builds fine from the command line.